global.R

```diff
@@ -10,25 +10,35 @@ library("wbstats")
 
 source("utils.R", local = T)
 
+download <- function() {
```

Comment:
Can we change this to downloadGithubData?
```suggestion
downloadGithubData <- function() {
```